workspaces/Cargo.toml

```diff
@@ -19,13 +19,15 @@ tokio = { version = "1", features = ["full"] }
 tokio-retry = "0.3"
 url = { version = "2.2.2", features = ["serde"] }
 
-near-crypto = { git = "https://github.com/near/nearcore", rev = "88a9e74d4b99867e417fe199cbd74559fd4fc349" }
-near-primitives = { git = "https://github.com/near/nearcore", rev="88a9e74d4b99867e417fe199cbd74559fd4fc349" }
-near-jsonrpc-primitives = { git = "https://github.com/near/nearcore", rev="88a9e74d4b99867e417fe199cbd74559fd4fc349" }
-
+near-account-id = "0.5"
+near-crypto = "0.5"
+near-primitives = "0.5"
```

Comment:
there is `0.10` version of this https://crates.io/crates/near-primitives/versions and I'm not sure the difference but we use this in the SDK so if we want to de-dup deps we might want to make sure it is synced.

Reply:
`near-jsonrpc-client` only accepts this specific version of those primitives, which is why I couldn't up them unless we want to create bridges from one version to the next